### PR TITLE
(PC-12727) fix(ReactNativeSreens): upgrade to 3.10.1 (latest) to remove console warning

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-12727

## Checklist

I have:

- [ ] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [ ] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|     |         |                 |                  |                  |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -13979,7 +13979,7 @@ react-native-safe-modules@^1.0.3:
   dependencies:
     dedent "^0.6.0"
 
-react-native-screens@^3.4.0:
+react-native-screens@^3.10.1:
```
Comment (kopax-polyconseil) [inline yarn.lock:13982]: bizarre, c'est déjà la version qui était installé

Comment (kopax-polyconseil) [reply yarn.lock:13982]: [`L13984`](https://github.com/pass-culture/pass-culture-app-native/pull/2466/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deL13984) n'a pas changé -_-

Comment (antoinewg) [reply yarn.lock:13982]: ah okey oui

**Comment by kopax-polyconseil** [comment]
Je merge pour avoir un package.json à jour, mais je dirais que le ticket est à delete ou a surveiller comme ceci ne corrige pas le warning, ce warning étant dans les libs qu’on utilise qui ne sont elle memes pas à jour.

**Pull request merged**